### ISSUE TITLE
How to train At?

### ISSUE THREAD

**Issue by ddghost** [Albert0147/G-SFDA#2]
Thanks for your interesting work.  I'm impressed with your method, but a little confused.
In the paper, "As and At are both trained on the source domain and are fixed during the adaptation to the target domain." How do you train At without target domain data?

**Comment by Albert0147** [comment]
As mentioned in Sec. 3.4: "first we train the model on Ds with the cross-entropy loss, with both source and target domain attention As, At, this is to provide a good initialization for target adaptation where only At is engaged".

The target adaptation must start from a good source model, which is the reason we train both As and At on source data at the beginning.
